Rakefile

```diff
@@ -1,60 +1,27 @@
-require 'cucumber/rake/task'
-
 namespace :cucumber do
 
   #task to launch registration tests
   task :register do
-    run_cucumber_task(@registration_task)
+    sh 'cucumber --tags @registration'
   end
```

Comment:
Hi @d-konovalov, thank you for the way you adviced in the previous comment)

It works but I took the liberty to change the approach and call `cucumber` with arguments directly using `sh` method
For me it does exactly the same as the version before, but `Rakefile` is kept clean without any helpers and declarations for tasks which are run then only once.

If in future will be needed to make this tasks more flexible by accepting argument for. ex. this could be easily extended using some `run_cucumber_with_args(args)` method which will accept args which are passed to ENV by Jenkins, for example

```Ruby
def run_cucumber_with_arg(arg)
  sh "cucumber #{arg}"
end
```
Something like this or *args version to accept multiple arguments.
Or cucumber profiles could be used in future too :)